### DISCUSSION TITLE
How to use the mac asset ? Not sure how to unarchive it or run it

### DISCUSSION THREAD

**Discussion by sharandec22** [lunasec-io/lunasec#339] [Help/Support] [answered]
Can someone help with how to run this on a mac?

**Comment by jakubskalak** [comment]
Just call
`sudo cp log4shell_1.4.0-log4shell_Darwin_x86_64 /usr/local/bin`

Then you can run:
`log4shell_1.4.0-log4shell_Darwin_x86_64 scan <path>`

**Reply by SW-41** [reply]
copied to usr/local/bin and when we try to run, get command not found.

**Reply by freeqaz** [reply]
You have to run `chmod +x ./log4shell` or `sudo chmod +x /usr/local/bin/log4shell` in order to mark the file as executable. None of us run Macs to test with (we just rented one in the cloud earlier), so I apologize for the confusion there!

**Reply by jakubskalak** [reply]
I made some tries to run it, so I must have changed chmod and forgot about that step.
Sorry 😄

**Comment by freeqaz** [comment, marked as answer]
We updated our docs on this, too, if you want to install it globally. I'm working on (hopefully) setting up Brew today. https://www.lunasec.io/docs/blog/log4j-zero-day-mitigation-guide/#automatically-scanning-your-package
